Nodejs/Tests/Core/Mocks/MockTextBuffer.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.NodejsTools;
+
+namespace NodejsTests.Mocks {
+    class MockTextBuffer : TestUtilities.Mocks.MockTextBuffer {

```

Comment:
Add a constructor which takes 2 arguments so people can specify the filename.

In the implementation generate the filename the same way the base class does when it's not specified, except with a js file.